repo/packages/E/elasticsearch/0/config.json

```diff
@@ -0,0 +1,41 @@
+{
+  "properties": {
+    "mesos": {
+      "depscription": "Mesos specific configuration properties",
+      "properties": {
+        "master": {
+          "default": "zk://master.mesos:2181/mesos",
+          "description": "The URL of the Mesos master. This should point to the Mesos ZNode on the Zookeeper machine.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "master"
+      ],
+      "type": "object"
+    },
+    "elasticsearch": {
```

Comment:
one of the needed properties is framework-name under the package name.

so:  elasticsearch.framework-name is the property.   It should be used for the marathon.json id value as well.